hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/tools/TestDFSAdmin.java

```diff
@@ -1377,7 +1377,8 @@ public Boolean get() {
         }
         scanIntoList(out, outs);
         scanIntoList(err, errs);
-        return !outs.isEmpty() && outs.get(0).contains("finished");
+        return !outs.isEmpty() && outs.stream().filter(line ->
+            line.contains("finished")).count() == 2;
```

Comment:
Can we explain this change?

Reply:
This test case is to refresh the configuration of two nodes in the cluster that are in the decommissioning state. It is necessary to wait for the 'finish' keyword of both nodes before the subsequent information verification becomes meaningful.